res/values/arrays.xml

```diff
@@ -85,6 +85,9 @@
 		<item>12</item> <item>13</item> <item>14</item> <item>15</item>
 	</string-array>
 
+	<string-array name="p_units_ev">
+		<item>1</item> <item>2</item>
```

Comment:
The values "1" and "2" aren't useful. You can replace them with "metric" and "imperial" or completely remove the array and use `getListItemIndex()` to determine the configured value.